src/code/z_actor.c

```diff
@@ -2226,14 +2242,14 @@ void Actor_DeactivateLens(PlayState* play) {
     }
 }
 
-void func_800B9120(ActorContext* actorCtx) {
-    s32 phi_v0 = CURRENT_DAY * 2;
+void Actor_InitHalfDaysBit(ActorContext* actorCtx) {
```

Comment:
I'm fine with `HalfDaysBit` but thoughts on possibly merging `mzx`'s suggestions and using `HalfDayOf`, with a comment that it is a bit field?